pack/pack.go

```diff
@@ -129,9 +182,9 @@ func (p *Pack) Resolve() (err error) {
 	reslv.Add("section", &p.Section)
 	reslv.Add("priority", &p.Priority)
 	reslv.Add("url", &p.Url)
-	reslv.AddList("depends", p.Depends)
-	reslv.AddList("optdepends", p.OptDepends)
-	reslv.AddList("makedepends", p.MakeDepends)
+	reslv.AddList("depends", dependsRaw)
+	reslv.AddList("optdepends", optDependsRaw)
+	reslv.AddList("makedepends", makeDependsRaw)
```

Comment:
This is most certainly wrong as we add empty lists. I don't understand what the resolver is used for, so I don't know how to handle it correctly.